smart_simulation/database/db_utils.py

```diff
@@ -0,0 +1,84 @@
+import logging
+import pathlib
+import sqlite3
+
+import pandas as pd
+
+import daiquiri
+from smart_simulation.cfg_templates.config import package_dir
+
+daiquiri.setup(
+    level=logging.INFO,
+    outputs=(
+        daiquiri.output.Stream(
+            formatter=daiquiri.formatter.ColorFormatter(
+                fmt="%(asctime)s [%(levelname)s] %(name)s.%(" "funcName)s: %(message)s"
+            )
+        ),
+    ),
+)
+
+PACKAGE_PATH = pathlib.Path(package_dir)
+LOCAL_PROD_DB = PACKAGE_PATH / "smart_simulation" / "database" / "production.db"
+
+
+def connect(database: pathlib.PurePath = LOCAL_PROD_DB) -> sqlite3.connect:
+    """
+    Connect to a sqlite3 database and return the connection.
+    Args:
+        database: Database to connect to, defaulted to local production database.
+
+    Returns: The database connection
+    """
+    if not isinstance(database, pathlib.PurePath):
+        logging.exception("weights_directory must be a pathlib Path.")
+        raise TypeError
+    if database.suffix != ".db":
+        logging.exception(f"Database: {database}, must be of extension type '.db'.")
```

Comment:
People will also end sqlite databases in `.sqlite`. I don't think there's a particular required format, but this may be just fine for your own purposes.